VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -294,9 +294,14 @@ private void checkNoResultView(EmptyQueryResultsFor query) {
         if (activity instanceof TaskProcessCallback) {
             if (discussionPostsAdapter.getCount() == 0) {
                 String resultsText = "";
+                boolean isAllPostsFilter = (postsFilter == DiscussionPostsFilter.ALL);
                 switch (query) {
                     case FOLLOWING:
-                        resultsText = getString(R.string.forum_no_results_for_following);
+                        if (!isAllPostsFilter) {
```

Comment:
Pessimist and Proud :sunglasses: 

![pessimist](https://cloud.githubusercontent.com/assets/1710804/14108437/d634d868-f5d6-11e5-9ce1-dddf88ea2e95.jpg)